### ISSUE TITLE
Bugfix: setup.py does not install data files

### ISSUE THREAD

**Issue by maddenp-cu** [dtcenter/METplus#2675]
## Describe the Problem ##

Required data files (e.g. `VERSION`) are not installed by `setup.py`, leading to a raised exception when running `run_metplus.py`.

### Expected Behavior ###

After `pip install`, `run_metplus.py` runs without raising an exception.

### Environment ###

Describe your runtime environment:

Ubuntu 22.04, Python 3.10.12, METplus 5.1.0

### To Reproduce ###

```
~ $ python3 -m venv demo
~ $ . demo/bin/activate
(demo) ~ $ pip install --quiet python-dateutil==2.8
(demo) ~ $ (cd METplus-5.1.0/ && pip install --quiet .)
(demo) ~ $ cp METplus-5.1.0/ush/*.py demo/bin
(demo) ~ $ run_metplus.py
Traceback (most recent call last):
  File "/home/maddenp/demo/bin/run_metplus.py", line 29, in <module>
    from metplus.util import metplus_check
  File "/home/maddenp/demo/lib/python3.10/site-packages/metplus/__init__.py", line 22, in <module>
    __version__ = get_metplus_version()
  File "/home/maddenp/demo/lib/python3.10/site-packages/metplus/__init__.py", line 4, in get_metplus_version
    return get_metplus_info('VERSION')
  File "/home/maddenp/demo/lib/python3.10/site-packages/metplus/__init__.py", line 18, in get_metplus_info
    with open(info_file, 'r') as file_handle:
FileNotFoundError: [Errno 2] No such file or directory: '/home/maddenp/demo/lib/python3.10/site-packages/metplus/VERSION'
```

Given the following patch:

```
diff --git a/setup.py b/setup.py
index 2114a77..142e2f6 100644
--- a/setup.py
+++ b/setup.py
@@ -49,6 +49,14 @@ setup(
          "Operating System :: OS Independent",
     ],
     python_requires=f'>={python_version_req}',
-    data_files=data_files,
+    include_package_data=True,
+    package_data={
+        "metplus": [
+            "PYTHON_VERSION",
+            "PYTHON_VERSION_MIN",
+            "RELEASE_DATE",
+            "VERSION",
+        ]
+    },
     zip_safe=False,
 )
```

I can get

```
~ $ python3 -m venv demo
~ $ . demo/bin/activate
(demo) ~ $ pip install --quiet python-dateutil==2.8
(demo) ~ $ (cd METplus-5.1.0/ && pip install --quiet .)
(demo) ~ $ cp METplus-5.1.0/ush/*.py demo/bin
(demo) ~ $ run_metplus.py

Usage: run_metplus.py arg1 arg2 arg3
    -h|--help               Display this usage statement

Arguments:
/path/to/parmfile.conf -- Specify custom configuration file to use
section.option=value -- override conf options on the command line
```

I've just started working with METplus but doubt that this is sufficient, as it ignores the rest of the contents of `data_files` as defined in `setup.py`, i.e. the files from the `parm/` directory. I followed the first sections of [these docs](https://setuptools.pypa.io/en/stable/userguide/datafiles.html) to get `VERSION` et al. installed, but maybe the [namespace packages](https://setuptools.pypa.io/en/stable/userguide/datafiles.html#subdirectory-for-data-files) method would help with getting files from `parm/`, which are not part of the `metplus` Python package, installed.

**Comment by georgemccabe** [comment]
Hi @maddenp-noaa,

I'm very sorry for the delayed response. Thank you for bringing this to our attention. Installing the METplus wrappers via pip is currently not **officially** supported.

The wrappers are typically "installed" by unpacking the release into a directory and (optionally) modifying `parm/metplus_config/defaults.conf` to set system-specific configurations. The `run_metplus.py` should be called from the ush directory to run the wrappers so it can find all of the files (parm, metplus/util, etc.) it needs to run.

Eventually we would like to be able to fully support installing and running the METplus wrappers as a python package, but there are a number of other changes that are needed to make this a reality. For example, it looks like installing non-package data files (like the files under parm) is longer supported. Some users have experimented with installing METplus so they can import utility functions in their own scripts.

I hope that helps clear things up. Please let me know if you have any questions or run into any issues running METplus wrappers.

**Comment by georgemccabe** [comment]
Hi @maddenp-noaa, PR #2699 includes some fixes to get the METplus wrappers to install as a python package properly to fix this bug. These changes were made into the develop branch, so they will be available in the v6.0.0-beta6 development release and v6.0.0 official release. We did not make these changes to the v5.1. We still recommend running METplus wrappers from the source code area, but these changes should be a step in the right direction towards running as an installed python package.
